python/audio_dsp/design/parse_json.py

```diff
@@ -330,10 +317,16 @@ def pipeline_to_dspjson(pipeline) -> DspJson:
             if op_type in ["PipelineStage", "DSPThreadStage"]:
                 continue
 
+            stage_in = []
+            for x in stage.i.edges:
+                if x.source is not None:
+                    stage_in.append([f"{x.source.label}", x.source_index])
+                else:
+                    stage_in.append([f"input", x.source_index])
```

Comment:
Should be inputs for consistency
```suggestion
                    stage_in.append([f"inputs", x.source_index])
```